node_modules/col-activities/tests/util.js

```diff
@@ -892,6 +892,40 @@ var assertEmailsForUsers = function(emails, users) {
 };
 
 /**
+ * Assert that a request to send weekly notification emails passes authorization but sends no messages
+ *
+ * @param  {RestClient}          client                           The REST client to make the request with
+ * @param  {Course}              course                           The Canvas course in which the user is interacting with the API
+ * @param  {Course}              dbCourse                         The database model for the course
+ * @param  {User[]}              users                            The users who should receive an email
+ * @param  {Function}            callback                         Standard callback function
+ * @param  {Object[]}            callback.emails                  The sent email messages
+ * @throws {AssertionError}                                       Error thrown when an assertion fails
+ */
+var assertSendWeeklyNotificationsEmpty = module.exports.assertSendWeeklyNotificationsEmpty = function(client, course, dbCourse, users, callback) {
```

Comment:
@johncrossman That's a good question. The convention in the codebase seems to be that these multipart assertions mostly live in `util.js` files, even if they're only used in one place. This allows the `test-*.js` files to function as slimmer descriptions of application flow (which is often fairly complex, since they're all integration tests).